pantheon-bundle/src/main/resources/SLING-INF/content/docs/modules/proc-search-view-results.adoc

```diff
@@ -3,14 +3,39 @@
 = Viewing search results
 
 [role="_abstract"]
-Search results are presented in a table organized in columns with the following headings:
+{ProductName} displays search results in two tables, *Modules* and *Assemblies*. which have the following columns:
 
-* *Title* - the content title
-* *Published* - the publication status, *Published* or *Not published*
-* *Draft uploaded* - the date the draft was uploaded
-* *Content Type* - the content type, *Assembly*, *Concept*, *Procedure*, or *Reference*
+* *Title*
+* *Repository* : the repository in which content is located
+* *Upload date* : the date on which the latest draft of the content was uploaded
+* *Last Published date* : the date the content was last published
 
-.Sorting search results
+You can select the number of results you want to be displayed per page.
 
-. Select a property from the fourth menu to sort according to that property (*Uploaded date*, *Title*, *Updated date*, or *Module type*)
-. Select the number of results you want to be displayed per page.
+
+== Viewing content details
+
+You can see additional details for each content item that is displayed in your search results.
+
+.Procedure
+
+In the Search results table, click the link of a content item.
++
+{ProductName} displays the following information:
+
+* *Module*: Complete directory path of the module's .adoc file.
+* *Product*
+* *Module type*
+* *First published date*
+* *Last published date*
+
+== Previewing text
+
+You can see the text of the latest draft of a module or an assembly using the {ProductName} the search engine.
```

Comment:
This statement does not sound very accurate to me. I would suggest something in the lines of 
"You can preview how the latest draft of a module or assembly will be rendered on the Customer Portal using {ProductName}"

Reply:
Good suggestion; thanks.